File: 2624-difference-between-element-sum-and-digit-sum-of-an-array/difference-between-element-sum-and-digit-sum-of-an-array.py
class Solution:
    def differenceOfSum(self, nums: List[int]) -> int:
        abs_val = sum(nums)
        deep_val = 0
        for i in range(len(nums)):
            dums = str(nums[i])
            for i in range(len(dums)):
                deep_val += int(dums[i])

        return(abs_val - deep_val)
                

File: 2624-difference-between-element-sum-and-digit-sum-of-an-array/difference-between-element-sum-and-digit-sum-of-an-array.cs
public class Solution {
    public int DifferenceOfSum(int[] nums) {

        int SumDigits = 0;
			int sumAllDigits = 0;
			foreach (var item in nums)
			{
				SumDigits += item;
				var temp = Convert.ToString(item);
				foreach (var i in temp)
				{
					sumAllDigits += i - '0';
				}
				
				
			}
			return(Math.Abs(SumDigits - sumAllDigits));







        
    }
}